src/showcase/src/pages/displayManage.astro

```diff
@@ -28,7 +30,7 @@ import Screen from "../layouts/Screen.astro";
             alias: "Yubikey Blue",
             remove: () => toast.info("remove"),
             rename: () => toast.info("rename"),
-            warn: "Something is rotten in the state of Device",
+            warn: html`Something is rotten in the state of Device`,
```

Comment:
This was a TS type issue that was not caught. `warn` expects a template, not a string.

Reply:
probably worth adding an `astro check` or something right? separately

Reply:
Yes, true, maybe.